File: php/src/vaas/Options/ForStreamOptions.php
<?php

namespace VaasSdk\Options;

class ForStreamOptions
{
    private const DEFAULT_TIMEOUT = 300;
    private const DEFAULT_REQUEST_ID = null;

    public function __construct(
        public bool $useHashLookup = true,
        public int $timeout = self::DEFAULT_TIMEOUT,
        public ?string $vaasRequestId = self::DEFAULT_REQUEST_ID) {}

    public static function fromVaasOptions(VaasOptions $options): self
    {
        return new self(
            $options->useCache,
            $options->timeout,
            self::DEFAULT_REQUEST_ID
        );
    }
}

File: python/src/vaas/options/for_stream_options.py
import uuid

class ForStreamOptions:
    def __init__(
            self,
            use_hash_lookup = True,
            vaas_request_id = None
    ):
        self.vaas_request_id = vaas_request_id
        self.use_hash_lookup = use_hash_lookup

    def from_vaas_config(self, vaas_options):
        self.use_hash_lookup = vaas_options.use_hash_lookup
        return ForStreamOptions()
